2-mandatory/2-financial-times.js

```diff
@@ -23,15 +50,45 @@ function titleWithFewestWords(allArticleTitles) {
     (Hint: remember that you can also loop through the characters of a string if you need to)
 */
 function headlinesWithNumbers(allArticleTitles) {
-    // TODO
+    const newarray=[];
+    const number=/[0-9]/;
+let i=0;
+ while (i < allArticleTitles.length) {
+   if (allArticleTitles[i].match(number)) {
+     newarray.push(allArticleTitles[i]);
+   }
+   i=i+1;
+ }
+return newarray;
 }
 
+
```

Comment:
Just as a nice to have: try to avoid committing unnecessary empty lines like this so we can keep the Pull request cleaner and avoid extra changes

Reply:
you are right, thanks for your advice